.buildkite/danger/Dangerfile

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+def release_branch?
+  danger.github.branch_for_base.start_with?('release/') || danger.github.branch_for_base.start_with?('hotfix/')
+end
+
+def main_branch?
+  danger.github.branch_for_base == 'trunk'
+end
+
+def wip_feature?
+  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
+  has_wip_title = github.pr_title.include?('WIP')
+
+  has_wip_label || has_wip_title
+end
+
+return if github.pr_labels.include?('Releases')
+
+github.dismiss_out_of_range_messages
+
+manifest_pr_checker.check_all_manifest_lock_updated
+
+labels_checker.check(
+  do_not_merge_labels: ['status: do not merge'],
+  required_labels: [//],
+  required_labels_error: 'PR requires at least one label.'
+)
+
+view_changes_need_screenshots.view_changes_need_screenshots
```

Comment:
Apologies if I missed this during code review—Seeing this method name in action made me realize it reads a bit awkward, especially in comparison with `pr_size_checker_check_diff_size` or `labels_checker_check`.

🤔 Not a big deal as long as the check work, but I wonder how we could rename it for consistency and readability.